mex/extractors/pipeline/wikidata.py

```diff
@@ -1,33 +1,20 @@
 from mex.common.exceptions import MExError
-from mex.common.models import ExtractedOrganization, ExtractedPrimarySource
-from mex.common.wikidata.extract import search_organization_by_label
-from mex.common.wikidata.models.organization import WikidataOrganization
-from mex.common.wikidata.transform import (
-    transform_wikidata_organizations_to_extracted_organizations,
+from mex.common.models import (
+    ExtractedOrganization,
 )
 from mex.extractors.pipeline import asset
-from mex.extractors.sinks import load
+from mex.extractors.wikidata.helpers import (
+    get_wikidata_organization_by_id,
+    get_wikidata_organization_ids_by_label,
+)
 
 
 @asset(group_name="default")
-def wikidata_organization_rki() -> WikidataOrganization:
-    """Extract WikidataOrganization for Robert Koch-Institut."""
-    if org := search_organization_by_label("Robert Koch-Institut"):
-        return org
-    msg = "RKI not found on wikidata, cannot proceed."
+def extracted_organization_rki() -> ExtractedOrganization:
+    """Extracts and Transforms RKI organization and loads result."""
+    wikidata_organization_ids_by_label = get_wikidata_organization_ids_by_label()
+    rki_id = wikidata_organization_ids_by_label["Robert Koch-Institut"]
+    if extracted_organization_rki := get_wikidata_organization_by_id(rki_id):
+        return extracted_organization_rki
```

Comment:
```suggestion
    if rki_id := wikidata_organization_ids_by_label.get("Robert Koch-Institut"):
        if extracted_organization_rki := get_wikidata_organization_by_id(rki_id):
            return extracted_organization_rki
```

just in case, someone (accidentally) removes RKI from the mapping